README.md

```diff
@@ -32,6 +32,7 @@ synchronisation between multiple hosts. A sample setup for this can be found [he
 
     © 2016 Christian Pointner <equinox@spreadspace.org>
     © 2016 Markus Grüneis <gimpf@gimpf.org>
+    © 2016 whawty contributors (see AUTHORS file)
```

Comment:
I like it but we should do this everywhere in the code - and there is no need for an exception for me and Markus.